spec/features/view_supplier_details_spec.rb

```diff
@@ -32,4 +32,11 @@
     expect(page).to have_content 'Pending'
     expect(page).to have_link 'Download submission file', href: download_url
   end
+
+  scenario 'lists the users linked to the supplier' do
+    FactoryBot.create(:user, suppliers: [supplier])
+
+    visit admin_supplier_path(supplier)
+    expect(page).to have_content 'Active?'
```

Comment:
To make this a better spec, it could also verify that some part of the user (their email?) actually appears on the page.